applications/ShapeOptimizationApplication/test_examples/04_Eigenfrequency_Maximization_3D_Cantilever/optimization_parameters.json

```diff
@@ -86,10 +86,10 @@
         "design_variables" : {
             "type"                               : "vertex_morphing",
             "filter" : {
-                "filter_function_type"       : "linear",
-                "filter_radius"              : 0.2,
-                "max_nodes_in_filter_radius" : 10000,
-                "matrix_free_filtering"      : true
+                "filter_function_type"        : "linear",
+                "filter_radius"               : 0.2,
+                "max_nodes_in_filter_radius"  : 10000,
+                "apply_matrix_free_filtering" : true
```

Comment:
i don`t think that the `apply` prefix is necessary, the fact that its a bool option, makes it clear already.